src/palace/manager/api/overdrive/api.py

```diff
@@ -772,31 +617,6 @@ def metadata_lookup(self, identifier: Identifier) -> dict[str, Any]:
         status_code, headers, content = self.get(url)
         return json.loads(content)  # type: ignore[no-any-return]
 
-    @classmethod
```

Comment:
`make_link_safe` had to get moved to a function outside of `OverdriveAPI` in order to break an import cycle.